pkg/minikube/assets/addons.go

```diff
@@ -902,6 +902,15 @@ func GenerateTemplateData(addon *Addon, cfg config.KubernetesConfig, netInfo Net
 			opts.Registries[name] = "" // Avoid nil access when rendering
 		}
 
+		// Without the line below, if you try to overwrite an image the default registry is still used in the templating
```

Comment:
Maybe: "Check if custom image is provided, if so, strip out default registry." as a first line for ease of reference.

Reply:
Added a concise first line